### PR TITLE
Better slack message

### LINKED ISSUE #28: Improve Slackbot Message
We currently use a very minimal Slackbot message.

## Before
Image 2020-01-21 at 2 49 14 PM

## Proposal 
I've extended one of Slacks official message templates. 
https://gist.github.com/benarent/e5368c8a1f867dc778dfa0095dd94137
image

### PR DESCRIPTION
**Pull request by DarthSim**
Closes #28 

Cluster name is not included since #27 is not resolved

### CODE REVIEW

File: access/slackbot/bot.go [hunk 1]
```diff
@@ -6,14 +6,20 @@ import (
...
 	"net/http"
-	"strings"
 
 	"github.com/gravitational/teleport-plugins/access"
 	"github.com/gravitational/trace"
 	"github.com/nlopes/slack"
 	log "github.com/sirupsen/logrus"
 )
 
+var emojiByStatus = map[string]string{
```
Comment (benarent) [inline access/slackbot/bot.go]: Solid Emoji selection.

**Comment by benarent** [comment]
@DarthSim   Can you please add a screenshot of how this looks in Slack?

**Comment by DarthSim** [comment]
There is another variant that is more compact: [link](https://api.slack.com/tools/block-kit-builder?mode=message&blocks=%5B%7B%22type%22%3A%22section%22%2C%22text%22%3A%7B%22type%22%3A%22mrkdwn%22%2C%22text%22%3A%22You%20have%20a%20new%20request%3A%22%7D%7D%2C%7B%22type%22%3A%22section%22%2C%22text%22%3A%7B%22type%22%3A%22mrkdwn%22%2C%22text%22%3A%22*ID*%3A%20495621b1-2782-48ff-b277-0bfc414c06fd%5Cn*User%3A*%20darthsim%5Cn*Role(s)%3A*%20admin%22%7D%7D%2C%7B%22type%22%3A%22context%22%2C%22elements%22%3A%5B%7B%22type%22%3A%22mrkdwn%22%2C%22text%22%3A%22*Status%3A*%20%3Ahourglass_flowing_sand%3A%20PENDING%22%7D%5D%7D%2C%7B%22type%22%3A%22actions%22%2C%22elements%22%3A%5B%7B%22type%22%3A%22button%22%2C%22text%22%3A%7B%22type%22%3A%22plain_text%22%2C%22emoji%22%3Atrue%2C%22text%22%3A%22Approve%22%7D%2C%22style%22%3A%22primary%22%2C%22value%22%3A%22click_me_123%22%7D%2C%7B%22type%22%3A%22button%22%2C%22text%22%3A%7B%22type%22%3A%22plain_text%22%2C%22emoji%22%3Atrue%2C%22text%22%3A%22Deny%22%7D%2C%22style%22%3A%22danger%22%2C%22value%22%3A%22click_me_123%22%7D%5D%7D%5D)

**Comment by xnutsive** [comment]
I've tested this — works as expected. Let's switch to a single column layout and merge this?

**Comment by DarthSim** [comment]
Rebased and changed message format to the compact one

**Pull request merged**